objects/rough-work/js/array-of-objects.js

```diff
@@ -0,0 +1,64 @@
+// 1. Create an array to hold quiz questions and answers
```

Comment:
I prefer to remove comments like this.
Per Clean Code:
> Comments are not maintainable and prone to lying.
Stop and try to express via code.

Reply:
Done